docker/alpine/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ocaml/opam2:alpine as bap-testing
+
+ENV HOME /home/opam
+WORKDIR $HOME
+
+RUN sudo apk update \
+ && sudo apk add \
```

Comment:
Why do we need to to install them manually? Depext should do it for us. If not, then please update the opam-repository files, so that depext will handle it.